pkg/orchestration/wiring/entangler.go

```diff
@@ -237,6 +243,9 @@ func (w *worker) entangle(resource *orch_v1.StateResource, stateMeta *meta_v1.Ob
 		exposedResources := make([]smith_v1.Resource, 0, len(res.WiringResult.Resources)) // optimistic allocation
 		for _, wiredResource := range res.WiringResult.Resources {
 			if wiredResource.Exposed {
+				w.logger.Warn(wiringResourcesDirectlyIsDeprecatedMsg,
```

Comment:
This will spam A LOT. For each bundle for each resource, basically. And quite often bundles are processed like 5 times in a second each. Because of self-induction effect. So, I'm not sure this is a good idea.